dodola/tests/test_services.py

```diff
@@ -636,6 +610,131 @@ def test_aiqpd_train(tmpdir, monkeypatch):
     assert aiqpd_model.af.shape == af_expected_shape
 
 
+def test_aiqpd_integration(tmpdir, monkeypatch):
+    """Tests that the shape of adjustment factors matches the expected shape"""
+    monkeypatch.setenv(
+        "HDF5_USE_FILE_LOCKING", "FALSE"
+    )  # Avoid thread lock conflicts with dask scheduler
+    lon = [-99.83, -99.32, -99.79, -99.23]
+    lat = [42.25, 42.21, 42.63, 42.59]
+    time = pd.date_range(start="1994-12-17", end="2015-01-15")
```

Comment:
As above, suggest removing this in favor of `xr.cftime_range`, then drop the pandas import and `from xclim.core.calendar import convert_calendar` above as well as the `convert_calendar` line below.

Reply:
Updated